packages/tools/src/isPackagerRunning.ts

```diff
@@ -16,18 +16,17 @@ import fetch from 'node-fetch';
  *   - `not_running`: the packager nor any process is running on the expected port.
  *   - `unrecognized`: one other process is running on the port we expect the packager to be running.
  */
-function isPackagerRunning(
+async function isPackagerRunning(
   packagerPort: string = process.env.RCT_METRO_PORT || '8081',
 ): Promise<'running' | 'not_running' | 'unrecognized'> {
-  return fetch(`http://localhost:${packagerPort}/status`).then(
-    res =>
-      res
-        .text()
-        .then(body =>
-          body === 'packager-status:running' ? 'running' : 'unrecognized',
-        ),
-    () => 'not_running',
-  );
+  try {
+    const result = await fetch(`http://localhost:${packagerPort}/status`);
+    const body = await result.text();
+
+    return body === 'packager-status:running' ? 'running' : 'unrecognized';
+  } catch (_error) {
+    return 'not_running';
+  }
```

Comment:
Feels like a good opportunity for enum here

Reply:
Yes, but the point of this PR was to configure Typescript ecosystem and prepare for migration, not refactoring every LOC.

Reply:
Well, the PR says "migrate", not "prepare for migration", so that's the reason I am asking. I'd love us to migrate to Typescript fully and embrace its features where possible. 

Shall we open a follow-up issue to track the backlog of things like this, and possibly others that you may already have in mind?

Reply:
Ye, sure.